ios/Exponent/Versioned/Core/Api/EXNotifications.m

```diff
@@ -306,6 +307,7 @@ - (instancetype)initWithExperienceId:(NSString *)experienceId kernelServiceDeleg
   UNNotificationCategory *newCategory = [UNNotificationCategory categoryWithIdentifier:[self internalIdForIdentifier:categoryId]
                                                                                actions:actionsArray
                                                                      intentIdentifiers:@[]
+                                                                     hiddenPreviewsBodyPlaceholder: previewPlaceholder
```

Comment:
Actually, I dug in to the Apple documentation (following my gut feeling) and it looks like this initializer (`+categoryWithIdentifier:actions:intentIdentifiers:hiddenPreviewsBodyPlaceholder:options:`) [has been introduced in iOS 11](https://developer.apple.com/documentation/usernotifications/unnotificationcategory/2873733-categorywithidentifier?language=objc), which means it won't be available on iOS 10 (on which the current one is used).

I think the solution would be to `if @available` the initializer and choose the proper one accordingly. Xcode should suggest a quick-fix solution. 🙂

Reply:
absolutely right, thanks for pointing that out 🙏 fixed it